src/main/resources/swagger/api-docs.yaml

```diff
@@ -2859,3 +2882,52 @@ definitions:
       billingAccount:
         type: string
         description: the id of the billing account to use, must start with 'billingAccounts/'
+
+  ErrorReport:
+    description: ''
+    required:
+      - source
+      - message
+      - causes
+      - stackTrace
+    properties:
+      source:
+        type: string
+        description: service causing error
+      message:
+        type: string
+        description: what went wrong
+      statusCode:
```

Comment:
I'm guessing it doesn't matter, but why is statusCode not required when source, message, causes, stackTrace all are?